doc/GuideForCommitters.md

```diff
@@ -26,6 +26,21 @@ follow when reviewing and merging pull requests and issues.  It also provides
 a checklist of items that committers must ensure have been completed prior to
 merging a pull request.
 
+## Committers and Areas of Expertise
+
+The following is an up to date list of the current committers within the project and their respective areas of expertise. It is encouraged to use this list as a guideline to request reviews on your pull requests from committers who is knowledgable in the area.
+
+| Committer          | Username       | Area of Expertise                               |
+|--------------------|----------------|-------------------------------------------------|
+| Andrew Young       | @youngar       | [cmake](https://github.com/eclipse/omr/labels/cmake), [jitbuilder](https://github.com/eclipse/omr/labels/jitbuilder), [port](https://github.com/eclipse/omr/labels/port)                         |
+| Angela Lin         | @alin00        |                                                 |
```

Comment:
I would remove Angela as she has asked to be removed from the committer list on the omr-dev mailing list but I have not figured out how to do it yet.

Reply:
Fixed in f52e03d